modal_proto/api.proto

```diff
@@ -1492,6 +1493,25 @@ message SandboxGetLogsRequest {
   string last_entry_id = 4;
 }
 
+message SandboxStdinWriteRequest {
```

Comment:
Created a separate [PR](https://github.com/modal-labs/modal-client/pull/1510) for proto changes